test/http/message.cpp

```diff
@@ -278,13 +278,27 @@ class message_test : public beast::unit_test::suite
         BEAST_EXPECT(m2.fields.exists("h"));
     }
 
+    void
+    testSpecialMembers()
+    {
+        response<string_body> r1;
+        response<string_body> r2{r1};
+        response<string_body> r3{std::move(r2)};
+        r2 = r3;
+        r1 = std::move(r2);
+        [r1]()
+        {
+        }();
```

Comment:
Is this intended as a compile-time test? Because it doesn't really _test_ anything beyond whether the special members are defined.